### DISCUSSION TITLE
Problem with screen turning off when PC is closed

### DISCUSSION THREAD

**Discussion by DJRoby19** [mathoudebine/turing-smart-screen-python#384] [General]
When I turn off the PC many times the 3.5 screen stays on
I use Linux mint turing smart 3.3.3 Python 3.10.12
What could it be?
It starts automatically when the PC is turned on but does not turn off

**Comment by mathoudebine** [comment]
Hi @DJRoby19 
How do you start the program automatically? Is it with a service, or a script?
Also, which version do you use?

**Reply by DJRoby19** [reply]
The program starts automatically as in your guide put in the opt folder with the old version I didn't have this problem
I use Linux Mint 21.2 Python 3.10.12
